wagtail/images/static_src/wagtailimages/scss/add-multiple.scss

```diff
@@ -124,6 +124,12 @@
         }
     }
 
+    .upload-duplicate {
+        .status-msg.duplicate {
```

Comment:
Why is this necessary? This feels like something that would be applicable to more than one message class.